src/components/TextField/TextField.tsx

```diff
@@ -43,6 +44,8 @@ export interface TPATextFieldProps extends TPAComponentProps {
   clearButtonAriaLabel?: string;
   /** Identifies the element that labels the clear button element. Optional. */
   clearButtonAriaLabelledby?: string;
+  /** Placement for Popover. Optional. */
+  placement?: Placement;
```

Comment:
Could you rename it for something more specific? It is not clear that it is only the placement of the error icon popover.

```suggestion
  errorTooltipPlacement?: Placement;
```

Reply:
And maybe you could add it right after the `errorTooltipMaxWidth` prop :)

Reply:
done :)